source/Government.cpp

```diff
@@ -206,15 +202,13 @@ void Government::Load(const DataNode &node)
 			for(const DataNode &grand : child)
 			{
 				if(grand.Size() >= 2)
-				{
-					const Government *gov = GameData::Governments().Get(grand.Token(0));
-					attitudeToward.resize(nextID, 0.);
```

Comment:
Instead of resizing the vector with `0.`, resize it with a sentinel values like NaN. Then, when checking the attitude you can determine whether the default attitude should be used or not.